inc/class-purger.php

```diff
@@ -70,7 +70,8 @@ public static function action_clean_post_cache( $post_id ) {
 		if ( $type && 'revision' === $type ) {
 			return;
 		}
-		pantheon_wp_clear_edge_keys( array( 'post-' . $post_id, 'rest-post-' . $post_id, 'post-huge', 'rest-post-huge' ) );
+    $keys = apply_filters( 'pantheon_purge_clean_post_cache', array( 'post-' . $post_id, 'rest-post-' . $post_id, 'post-huge', 'rest-post-huge' ), $post_id );
```

Comment:
For each filter, can you make sure you're indenting with tabs, not spaces, and add the appropriate doc block?

Reply:
I'll do.